app/celery/provider_tasks.py

```diff
@@ -92,11 +92,15 @@ def send_sms_to_provider(self, service_id, notification_id):
         except SmsClientException as e:
             try:
                 current_app.logger.error(
-                    "SMS notification {} failed".format(notification_id)
+                    "RETRY: SMS notification {} failed".format(notification_id)
```

Comment:
I think if it throws an exception it is a retry (it's going to) which is worth knowing about.